containerd-shim-v2/service.go

```diff
@@ -43,6 +43,8 @@ const (
 
 	chSize      = 128
 	exitCode255 = 255
+
+	pubTimeOut = 5 * time.Second
```

Comment:
Thanks. A couple of nits:

- It would be useful to explain "why 5?" in a comment here though.
- I'd consider calling this `publishTimeout` as "pub" is ambiguous - it might imply "public" or even "[public house](https://en.wikipedia.org/wiki/Pub)" if you live in certain parts of the world :laughing: